public/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Real Euroleague Standings",
+  "short_name": "Real EL Standings",
+  "description": "Real-time Euroleague basketball standings and statistics",
+  "start_url": "/",
+  "display": "standalone",
+  "background_color": "#ffffff",
+  "theme_color": "#000000",
+  "icons": [
+    {
+      "src": "/euroleague.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "/euroleague.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    },
```

Comment:
these two icons can be removed, the last one is enough because it contains correct size.. manifest can contain multiple images, but then they need to be different in size and file name..